src/components/Header.tsx

```diff
@@ -0,0 +1,17 @@
+import LoginLogo from './loginLogo';
+import Logo from './Logo';
+import NavLinks from './NavLinks';
+
+const Header = () => {
+  return (
+    <header className="header-container">
+      <Logo />
+      <div className="header-right">
```

Comment:
```suggestion
      <Box className="header-right">
```
i would try to use MUI components